src/OSPSuite.Presentation/Binders/ObservedDataDragDropBinder.cs

```diff
@@ -53,26 +53,23 @@ public IEnumerable<DataRepository> DroppedObservedDataFrom(DragEventArgs e)
          return getObservedDataNodesFrom(e).Select(observedDataNode => observedDataNode.Tag.Subject);
       }
 
-      private IEnumerable<ObservedDataNode> getObservedDataNodesFrom(DragEventArgs e)
+      private IReadOnlyList<ObservedDataNode> getObservedDataNodesFrom(DragEventArgs e)
       {
          var dataNodes = e.Data<IEnumerable<ITreeNode>>();
-         if (dataNodes == null) return Enumerable.Empty<ObservedDataNode>();
+         if (dataNodes == null)
+            return new List<ObservedDataNode>();
 
          var treeNodes = dataNodes as IList<ITreeNode> ?? dataNodes.ToList();
          if (areAllObservedDataNodes(treeNodes))
-         {
             return observedDataNodesFromObservedDataNodes(treeNodes);
-         }
 
          if (areAllObservedDataClassificationNodes(treeNodes) || areAllRootNodes(treeNodes))
-         {
             return observedDataNodesFromClassificationNodes(treeNodes);
-         }
 
-         return Enumerable.Empty<ObservedDataNode>();
+         return new List<ObservedDataNode>();
       }
 
-      private IEnumerable<ObservedDataNode> observedDataNodesFromClassificationNodes(IEnumerable<ITreeNode> treeNodes)
+      private IReadOnlyList<ObservedDataNode> observedDataNodesFromClassificationNodes(IEnumerable<ITreeNode> treeNodes)
```

Comment:
I changed that because now I am using the fact that it is an IReadOnlyLust to do stuff in PKSim. Because it was not return like that, I had to call ToList all the time